rust/README.md

```diff
@@ -0,0 +1,88 @@
+
+# Peaq Network CharmEV Rust API Library
+
+#### Introduction
+This library handles the core api features of:
+* interacting with the node
+* connecting to provider peer network
```

Comment:
remove word network as it can be misleading

Reply:
corrected